src/mlpack/tests/main_tests/emst_test.cpp

```diff
@@ -106,10 +104,10 @@ TEST_CASE_METHOD(EMSTTestFixture, "EMSTFirstTwoOutputRowsIntegerTest",
   for (size_t i = 0; i < params.Get<arma::mat>("output").n_cols; ++i)
   {
     REQUIRE(params.Get<arma::mat>("output")(0, i) ==
-        Approx(boost::math::iround(params.Get<arma::mat>("output")(0, i))).
+        Approx(static_cast<int>(std::round(params.Get<arma::mat>("output")(0, i)))).
         epsilon(1e-7));
     REQUIRE(params.Get<arma::mat>("output")(1, i) ==
-        Approx(boost::math::iround(params.Get<arma::mat>("output")(1, i))).
+        Approx(static_cast<int>(std::round(params.Get<arma::mat>("output")(1, i)))).
         epsilon(1e-7));
```

Comment:
is there any reason here to do the static casting?

Reply:
std::round gives double as output. So I just took static_cast to int.

Reply:
??

Reply:
I think it's fine to just leave the result as `double`---I think `Approx()`'s constructor will expect a floating-point argument anyway.

Reply:
Done.